phpseclib/Crypt/RSA.php

```diff
@@ -2464,7 +2464,7 @@ function _emsa_pss_encode($m, $emBits)
         // be output.
 
         $emLen = ($emBits + 1) >> 3; // ie. ceil($emBits / 8)
-        $sLen = $this->sLen == false ? $this->hLen : $this->sLen;
+        $sLen = $this->sLen === false ? $this->hLen : $this->sLen;
```

Comment:
One of our users reported at #768 that this may have actually broken functionality. It sounds like, logically, this should have been `$sLen = $this->sLen ? $this->sLen : $this->hLen;` instead. Note that s and h was swapped.